assets/wizards/readerRevenue/views/donation/index.js

```diff
@@ -46,7 +46,7 @@ class Donation extends Component {
 						checked={ tiered }
 						onChange={ tiered => onChange( { ...data, tiered } ) }
 					/>
```

Comment:
On Mobile (iPhone XS) the Donation price elements break:

<img width="1595" alt="Screen Shot 2019-12-11 at 11 11 58 PM" src="https://user-images.githubusercontent.com/1477002/70682169-e8246880-1c6b-11ea-9bb2-8d2d37cb258d.png">

Reply:
Should be fixed in #329